apps/wpcom-block-editor/src/wpcom/features/live-preview/hooks/use-can-preview-but-need-upgrade.ts

```diff
@@ -58,10 +67,16 @@ export const useCanPreviewButNeedUpgrade = ( {
 	 * Get the theme and site info to decide whether the user needs to upgrade the plan.
 	 */
 	useEffect( () => {
+		const livePreviewUpgradeTypes = [ WOOCOMMERCE_THEME, PREMIUM_THEME ];
+
+		if ( config.isEnabled( 'themes/tiers' ) ) {
+			livePreviewUpgradeTypes.push( PERSONAL_THEME );
+		}
+
 		/**
 		 * Currently, Live Preview only supports upgrades for WooCommerce and Premium themes.
 		 */
-		if ( previewingTheme.type !== WOOCOMMERCE_THEME && previewingTheme.type !== PREMIUM_THEME ) {
+		if ( previewingTheme?.type && livePreviewUpgradeTypes.includes( previewingTheme.type ) ) {
```

Comment:
Hi @rcrdortiz! CC: @dsas 
This condition should be the opposite (`! livePreviewUpgradeTypes.includes`), as we need to show the upgrade notice for Premium/Woo themes.

I fixed it in https://github.com/Automattic/wp-calypso/pull/85771/.